### PR TITLE
07 io streams

### PR DESCRIPTION
**Pull request by SlavaPeshko**

### CODE REVIEW

**Comment by SlavaPeshko** [comment]
finish

File: 07-IO Streams/IOStreams/TestTasks.cs [hunk 1]
```diff
@@ -34,7 +34,30 @@ public static IEnumerable<PlanetInfo> ReadPlanetInfoFromXlsx(string xlsxFileName
...
+
+			using (var pac = Package.Open(xlsxFileName))
+			{
+				var uri1 = new Uri("/xl/worksheets/sheet1.xml", UriKind.Relative);
+				var uri2 = new Uri("/xl/sharedStrings.xml", UriKind.Relative);
+
+				var part = pac.GetPart(uri1);
+				var part2 = pac.GetPart(uri2);
+
+				var docNamesPlanet = XDocument.Load(part2.GetStream());
```
Comment (AlexMokin) [inline 07-IO Streams/IOStreams/TestTasks.cs]: don't forget to despose of streams

Comment (AlexMokin) [reply 07-IO Streams/IOStreams/TestTasks.cs]: also in English noun modifiers go before the noun:

-  variable contains the doc that contains names => `names` is a modifier, the `doc` is a noun => namesDoc.

- names of the planet => `planet` is a modifier, `names`  is a noun,

So the more readable name is `planetNamesDoc` - the doc that contains the names of the planets

File: 07-IO Streams/IOStreams/TestTasks.cs [hunk 2]
```diff
@@ -34,7 +34,30 @@ public static IEnumerable<PlanetInfo> ReadPlanetInfoFromXlsx(string xlsxFileName
...
+				var ns = docNamesPlanet.Root.GetDefaultNamespace();
+
+				var namesPalnet = docNamesPlanet.Root.Descendants(ns + "t").Select(n => n.Value);
```
Comment (AlexMokin) [inline 07-IO Streams/IOStreams/TestTasks.cs]: same as above, you have the names of the planet => planet names

File: 07-IO Streams/IOStreams/TestTasks.cs (hunk 1 above)
Comment (AlexMokin) [reply 07-IO Streams/IOStreams/TestTasks.cs]: https://learnenglish.britishcouncil.org/en/english-grammar/adjectives/noun-modifiers

**Comment by SlavaPeshko** [comment]
updated

**Comment by AlexMokin** [comment]
very good !

**Pull request closed without merging**